recipes/cctbx/LICENSE.txt

```diff
@@ -0,0 +1,44 @@
+*** License agreement ***
```

Comment:
No need for this file, conda build will find it from the source tarball.

Reply:
Just committed a change.